app/lib/screens/medical/manager/history/history_manager.dart

```diff
@@ -0,0 +1,168 @@
+import 'dart:convert';
+import 'package:flutter/material.dart';
+import 'package:flutter_flavor/flutter_flavor.dart';
+import 'package:flutter_secure_storage/flutter_secure_storage.dart';
+import 'package:http/http.dart' as http;
+import 'package:ira/screens/medical/manager/history/history_manager_detail.dart';
+import 'package:paginated_search_bar/paginated_search_bar.dart';
+
+class HistoryManager extends StatefulWidget {
+  const HistoryManager({Key? key}) : super(key: key);
+  @override
+  State<HistoryManager> createState() => _HistoryManagerState();
+}
+
+class _HistoryManagerState extends State<HistoryManager> {
+  final secureStorage = const FlutterSecureStorage();
+  String baseUrl = FlavorConfig.instance.variables['baseUrl'];
+  List<StudentModel> students = [];
+
+  Future<List<StudentModel>> _getPatientData(String email) async {
+    String? token = await secureStorage.read(key: 'staffToken');
+    final requestUrl = Uri.parse(
+      baseUrl + '/medical/search/patient?email=$email',
+    );
+
+    final Map<String, String> headers = {
+      "Content-Type": "application/x-www-form-urlencoded",
+      'Authorization': token != null ? 'Token ' + token : '',
+    };
+
+    try {
+      final response = await http.get(requestUrl, headers: headers);
+
+      final responseData = json.decode(response.body);
+      final List<StudentModel> studentsData = [];
+      responseData.forEach((student) {
+        studentsData.add(StudentModel.fromJson(student));
+      });
+
+      if (response.statusCode == 200) {
+        return studentsData;
+      } else {
+        return [];
+      }
```

Comment:
The medical manager should be able to search using entry number, not email.

Reply:
Medical managers can search using the name. But email is required to fetch list of patients